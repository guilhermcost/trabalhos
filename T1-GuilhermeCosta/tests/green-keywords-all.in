const
if 
else
int 
void 
return
while 
for
