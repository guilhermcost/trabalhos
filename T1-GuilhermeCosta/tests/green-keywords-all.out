(1,KEY,"const")
(2,KEY,"if")
(3,KEY,"else")
(4,KEY,"int")
(5,KEY,"void")
(6,KEY,"return")
(7,KEY,"while")
(8,KEY,"for")
